lib/LedgerNodeBlocks.js

```diff
@@ -51,6 +70,29 @@ module.exports = class LedgerNodeBlocks {
     return this.storage.blocks.getLatest();
   }
 
+  /**
+   * Gets the latest block height for the ledger.
+   *
+   * @param {object} [options] - The options to use.
+   * @param {Boolean} [options.useCache=false] - Use the in memory cache.
+   *
+   * @return {Promise<Number>} the latest block height for the ledger.
+   */
+  async getLatestBlockHeight({useCache = false} = {}) {
+    const {ledgerNodeId} = this;
+    let basisBlockHeight;
+    if(useCache) {
+      basisBlockHeight = blockHeightCache.get(ledgerNodeId);
+    }
+    if(basisBlockHeight !== undefined) {
```

Comment:
This basisBlockHeight check block could be moved up inside the useCache check block.

Reply:
Done.